2017-04-24 4:16:28 Last error : exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Error: Call to private method AppBundle\Entity\Commande::setIdCommande() from context 'AppBundle\Controller\ClientController'' in C:\Users\alinux\Documents\session4\progWeb\TP\cegep_session4_progWeb_TP\src\AppBundle\Controller\ClientController.php:221
Stack trace:
#0 C:\Users\alinux\Documents\session4\progWeb\TP\cegep_session4_progWeb_TP\src\AppBundle\Controller\ClientController.php(192): AppBundle\Controller\ClientController->retrieveCommandesFromUser()
#1 C:\Users\alinux\Documents\session4\progWeb\TP\cegep_session4_progWeb_TP\var\cache\dev\classes.php(4217): AppBundle\Controller\ClientController->commandesAction()
#2 C:\Users\alinux\Documents\session4\progWeb\TP\cegep_session4_progWeb_TP\var\cache\dev\classes.php(4217): call_user_func_array:{C:\Users\alinux\Documents\session4\progWeb\TP\cegep_session4_progWeb_TP\var\cache\dev\classes.php:4217}()
#3 C:\Users\alinux\Documents\session4\progWeb\TP\cegep_session4_progWeb_TP\var\cache\dev\classes.php(4172): Symfony\Component\HttpKernel\HttpKernel->handleRaw()
#4 C:\Users\alinux\Documents\session4\progWeb\TP\cegep_session4_progWeb_TP\vendor\symfony\symfony\src\Symfony\Component\HttpKernel\Kernel.php(168): Symfony\Component\HttpKernel\HttpKernel->handle()
#5 C:\Users\alinux\Documents\session4\progWeb\TP\cegep_session4_progWeb_TP\web\app_dev.php(28): Symfony\Component\HttpKernel\Kernel->handle()
#6 C:\Users\alinux\Documents\session4\progWeb\TP\cegep_session4_progWeb_TP\vendor\symfony\symfony\src\Symfony\Bundle\FrameworkBundle\Resources\config\router_dev.php(40): ()
#7 C:\Users\alinux\Documents\session4\progWeb\TP\cegep_session4_progWeb_TP\vendor\symfony\symfony\src\Symfony\Bundle\FrameworkBundle\Resources\config\router_dev.php(0): {main}()
#8 {main}